app/controllers/bright_events_controller.rb

```diff
@@ -0,0 +1,58 @@
+class BrightEventsController < ApplicationController
+  before_action :find_event, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @events = BrightEvent.all
+  end
+
+  def show
+    @event = find_event
+  end
```

Comment:
You don't need to call the find_event method. It is called and executed before any action for :show, :edit, :update and :destroy

Reply:
Done :+1: